static/skin/taskbar.js

```diff
@@ -7,7 +7,14 @@ const jq = jQuery.noConflict(true);
 jq(document).ready(() => {
     (function ($) {
         const root = $( `link[type='root']` ).attr("href");
-        const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
+        const urlParams = new URLSearchParams(window.location.search);
+
+        content = window.location.pathname.split(`${root}/`)[1].split('/')[0];
+        if (window.location.href.search('\\/search\\?content\\=') != -1) {
```

Comment:
This fix pushes the imaginary corner case discussed in the previous review further but doesn't eliminate it. Suppose that there is a ZIM archive with an article named `search`. Besides, this fix will break if the `content` parameter stops being the first parameter in the query.

Why do you search for `search`?  Why don't you simply check that the path of the url is `${root}/search`?

Reply:
That will be better and robust! JS is something that I am not very familiar with, but I should be able to catch up soon. I'll make the change.